src/wallet/spend.cpp

```diff
@@ -849,7 +849,7 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     }
     if (feeCalc.reason == FeeReason::FALLBACK && !wallet.m_allow_fallback_fee) {
         // eventually allow a fallback fee
-        return util::Error{_("Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee.")};
+        return util::Error{strprintf(_("Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable %s."), "-fallbackfee")};
```

Comment:
Strictly speaking, the middle sentence is a bit redundant (and also contains the option name)

Reply:
For translators, it will be better to drop the middle sentence or put a place holder instead of the option name.